zproject/default_settings.py

```diff
@@ -629,3 +629,8 @@
 SIGNED_ACCESS_TOKEN_VALIDITY_IN_SECONDS = 60
 
 CUSTOM_AUTHENTICATION_WRAPPER_FUNCTION: Optional[Callable[..., Optional["UserProfile"]]] = None
+
+# The max connect minutes of smtp to reopen connect
+# when the value more than 0 , smtp connect will reopen after
+# minutes of value
+SMTP_MAX_CONNECTION_MINUTES = 0
```

Comment:
Please clean up the typography here. But also, should `None` (no maximum) be a supported value distinct from 0?

Reply:
Picked this directly from #24896 will do the needful

I think `None` can be a supported value because all the checks are done treating SMTP_MAX_CONNECTION_MINUTES as bool so it wouldn't make any difference.

Reply:
No, this is wrong.

0 should mean "the max time to keep this open is 0" which means that every time we use the connection, we re-open it.  Effectively, we always close the socket after using it, and do not use a persistent connection.  Arguably we should make the code special-case this value to immediately close the connection after use.

`None` should mean "there is no max time" which means that we _never_ close the connection, and _always_ attempt to reuse it.

Those two values mean different things.